package.json

```diff
@@ -32,48 +32,50 @@
     "format": "npx prettier --write \"**/*.{js,ts,tsx,json}\""
   },
   "devDependencies": {
+    "@reduxjs/toolkit": "^2.2.6",
     "@types/cmd-shim": "^5.0.2",
     "@types/is-running": "^2.1.2",
     "@types/node": "20.14.9",
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
     "@types/redux-logger": "^3.0.13",
+    "@types/semver": "^7.5.8",
     "@typescript-eslint/eslint-plugin": "7.15.0",
+    "@vitejs/plugin-react": "^4.3.1",
+    "classnames": "^2.5.1",
     "cross-env": "7.0.3",
+    "decentraland-ui2": "^0.1.0",
     "electron": "31.1.0",
     "electron-builder": "24.13.3",
     "eslint": "8.57.0",
     "eslint-plugin-vue": "9.27.0",
+    "flat": "^6.0.1",
     "happy-dom": "14.12.3",
     "nano-staged": "0.8.0",
     "playwright": "1.45.0",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-intl": "^6.6.8",
+    "react-redux": "^9.1.2",
+    "react-router-dom": "^6.24.1",
+    "redux-logger": "^3.0.6",
```

Comment:
I moved all the `renderer` dependencies to the `devDependencies`, since they are not required on the `main` process. This is to make the `npm install` as fast as possible for the users. 

We should try to keep dependencies this way (if only used by the renderer -> devDependency) to impact as little as possible on the time it takes to install the Editor.